code/modules/clothing/masks/gasmask.dm

```diff
@@ -18,25 +18,36 @@
 
 /obj/item/clothing/mask/gas/attackby(obj/item/W,mob/user)
 	..()
+	if(type != "/obj/item/clothing/mask/gas")
```

Comment:
Actually I know I told you to do this but you might want to test it.
It might need to be:
`if("[type]" != "typepath goes here")
Or if it doesn't work you could try to test it without using strings at all